nuget/Vcpkg.Nuget.props

```diff
@@ -5,6 +5,9 @@
       AssemblyFile="$(MsbuildThisFileDirectory)VcpkgBuildTask.dll"/>
 
   <Target Name="Vcpkg" BeforeTargets="PreBuildEvent">
+    <PropertyGroup>
+      <VcpkgTaskTimeout Condition="'$(VcpkgTaskTimeout)' == '' ">600000</VcpkgTaskTimeout>
```

Comment:
this is a great solution - want to make sure `VcpkgTaskTimeout` ends up in the README as well.

Reply:
In my testing I also found out that the order in which the statements appear in the project file have an impact on it failing. Can adapt the README as well.